Makefile

```diff
@@ -35,18 +36,18 @@ test: | mocks	## Run code tests
 clean-mocks:	## Remove mocks directory
 	rm -rf mocks
 
-build-controller-image:	## Build container image for SERVICE
-	./scripts/build-controller-image.sh -s $(SERVICE)
+build-controller-image:	## Build container image for AWS_SERVICE
```

Comment:
Need to change this back to SERVICE since this is output for `make help` (and the variable to set for the user is SERVICE, not AWS_SERVICE)

Reply:
Haha, I would like to blame `Find and Replace` functionality.
I fixed it, PTAL. Thanks.